Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM alpine:3.5
+
+RUN apk add --no-cache python py-pip ca-certificates && \
+    pip install --upgrade setuptools
+
+ADD . /src
+
+WORKDIR /src
+
+RUN python setup.py install
+
+ENTRYPOINT ["gixy"]
```

Comment:
Added.

If you add a .editorconfig to the root of the project, those kind of conventions can be automatically applied by the editor. 👍